GSoC25/EntityLinking/methods.py

```
@@ -0,0 +1,147 @@
+def EL_DBpedia_lookup(query: str, max_results: int):
+    """Perform entity linking using DBpedia lookup service on
+    the given query.
+
+    Args:
+        query (str): The entity mention that you want to link to a resource in dbpedia.
+        max_results (int): The maximum candidate entities/resources that you want to fetch.
+
+    Returns:
+        List: A list of the candidate entities/resources.
+    """
+    response = requests.post(
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)